app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -118,8 +123,13 @@
         reportType = org.acra.sender.HttpSender.Type.JSON,
         httpMethod = org.acra.sender.HttpSender.Method.PUT)
 public class CommCareApplication extends Application {
+
     private static final String TAG = CommCareApplication.class.getSimpleName();
 
+    // Tracking ids for Google Analytics
+    private static final String LIVE_TRACKING_ID = "UA-69708208-1";

```

Comment:
If it is alright to have just on release builds then you would add it to /home/jenkins/.gradle/gradle.properties on the jenkins mobile server. Then you would have to wire things up in similar ways to how ACRA and other keys are setup: https://github.com/dimagi/commcare-odk/blob/master/build.gradle#L77-L79